.travis.yml

```diff
@@ -13,25 +12,22 @@ env:
   - DB=Firebird 
   - DB=MySQL CONNECTION_STRING="Server=127.0.0.1;Uid=root;Database=nhibernate;Old Guids=True;"   
   - DB=SQLite 
-matrix:
-  allow_failures:
-    - env: DB=MySQL CONNECTION_STRING="Server=127.0.0.1;Uid=root;Database=nhibernate;Old Guids=True;"
 before_install:
-  - curl https://packages.microsoft.com/keys/microsoft.asc | sudo apt-key add -
-  - curl https://packages.microsoft.com/config/ubuntu/14.04/prod.list | sudo tee /etc/apt/sources.list.d/microsoft.list
   - sudo apt-get update -qq
   - sudo apt-get install -y powershell 
   - |-
     if [[ "$DB" == "MySQL" ]]
     then
-      echo -e '[server]\nlower_case_table_names=1' | sudo tee -a /etc/mysql/my.cnf
+      echo -e '[server]\nlower_case_table_names=1\n[mysqld]\ncharacter-set-server=utf8\ncollation-server=utf8_general_ci' | sudo tee -a /etc/mysql/my.cnf
       sudo service mysql restart
     fi
   - |-
     if [[ "$DB" == "Firebird" ]]
     then
-      sudo apt-get install -y libicu-dev libtommath-dev curl
-      curl -L -O https://github.com/FirebirdSQL/firebird/releases/download/R3_0_5/Firebird-3.0.5.33220-0.amd64.tar.gz
+      sudo apt-get install -y libtommath-dev
+      # This would be required on bionic and above
+      # sudo ln -s /usr/lib/x86_64-linux-gnu/libtommath.so.1 /usr/lib/x86_64-linux-gnu/libtommath.so.0
+      wget -q https://github.com/FirebirdSQL/firebird/releases/download/R3_0_5/Firebird-3.0.5.33220-0.amd64.tar.gz
```

Comment:
`curl -L -O URL` and `wget URL` do exactly same. I decided to leave `wget` as it is a standard GNU utility.